docs/Initial-Setup/Setting-up-your-Manager-Instance.rst

```diff
@@ -22,7 +22,7 @@ To launch a manager instance, follow these steps:
    data is preserved when you stop/start the instance, and your data is
    not lost when pricing spikes on the spot market.
 2. When prompted to select an AMI, search in the ``Community AMIs`` tab for
-   "FPGA" and select the option that starts with ``FPGA Developer AMI - 1.3.5``.
+   "FPGA" and select the option that starts with ``FPGA Developer AMI - 1.4.0``.
```

Comment:
Would it be easier just to check for this?

Reply:
I think the aws-fpga scripts already check, at least for this change.